### PR TITLE
Merge with upstream

### PR DESCRIPTION
**Pull request by devtekve**
This is bringing cereal up to speed with the latest master changes

### CODE REVIEW

File: log.capnp [hunk 1]
```diff
@@ -2316,4 +2320,4 @@ struct Event {
     driverStateDEPRECATED @59 :DriverStateDEPRECATED;
     sensorEventsDEPRECATED @11 :List(SensorEventData);
   }
-}
+}
```
Comment (sunnyhaibin) [inline log.capnp:2323]:
```suggestion
}

```

File: car.capnp [hunk 2]
```diff
@@ -312,7 +312,7 @@ struct CarState {
 
   # deprecated
   errorsDEPRECATED @0 :List(CarEvent.EventName);
-  brakeLights @19 :Bool;
+  brakeLightsDEPRECATED @19 :Bool;
```
Comment (sunnyhaibin) [inline car.capnp:315]: Let's preserve this for now.
```suggestion
  brakeLights @19 :Bool;
```

Comment (devtekve) [reply car.capnp:315]: This is intentional for the refactor, it's already taken care of on the sync

Comment (sunnyhaibin) [reply car.capnp:315]: Ah gotcha, didn't see you updated in the other places. too.

**Pull request merged**